CHANGELOG.md

```diff
@@ -132,6 +132,8 @@ to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 * `jj resolve` now accepts new built-in merge tools `:ours` and `:theirs`.
   These merge tools accept side #1 and side #2 of the conflict respectively.
 
+* `jj show` patches can now be suppressed with `--no-patch`.
```

Comment:
reminder: you'll need to relocate the changelog entry.

We'll have a point release today or tomorrow, so please hold this change until then.